pkg/operator/configobservation/network/observe_network.go

```diff
@@ -65,35 +58,38 @@ func ObserveRestrictedCIDRs(genericListers configobserver.Listers, recorder even
 	}
 
 	if err := unstructured.SetNestedStringSlice(admissionControllerConfig.Object, restrictedCIDRs, "restrictedCIDRs"); err != nil {
-		errs = append(errs, err)
+		return existingConfig, append(errs, err)
 	}
 
 	observedConfig := map[string]interface{}{}
-	unstructured.SetNestedMap(observedConfig, admissionControllerConfig.Object, configPath...)
+	if err := unstructured.SetNestedMap(observedConfig, admissionControllerConfig.Object, configPath...); err != nil {
+		return existingConfig, append(errs, err)
```

Comment:
we should behave consistently, for example, won't return an error in a similar case https://github.com/openshift/cluster-kube-apiserver-operator/blob/423e7c85c9aafa12a4e5981baa2970f85b46ed22/pkg/operator/configobservation/images/observe_images.go#L43